sympy/ntheory/generate.py

```diff
@@ -14,13 +14,24 @@
 from sympy.core.compatibility import as_int, range
 
 
+def _azeros(n):
+    return _array('l', [0]*n)
+
+
 def _arange(a, b):
-    ar = _array('l', [0]*(b - a))
+    ar = _azeros(b - a)
     for i, e in enumerate(range(a, b)):
         ar[i] = e
     return ar
 
 
+def _aset(*v):
+    ar = _azeros(len(v))
```

Comment:
Could this be `ar = _array('l', v)`?

Reply:
Yes, I will make the changes.

Reply:
I wanted to get away from creating the `_array` in more than one place. The name ('l' or 'i') has to match. By localizing this in `_azeros` there is only one place that the array is created.